cli/azd/cmd/testdata/TestUsage-azd-template-source.snap

```diff
@@ -8,7 +8,14 @@ Usage
   azd template source [command]
 
 Available Commands
-  add   	: Adds an azd template source at the specified key (Beta)
+  add   	: Adds an azd template source at the provided key (Beta)
+  Use known-keys w/o a type for:
+   ・default: azd built-in template list
+   ・awesome-azd: Includes all templates from http://aka.ms/awesome-azd
+  For custom key, use type and location flags. Supported types:
+   ・file: local file only 
+   ・url: public urls, only, to download json source
+   ・gh: path to json file in a GitHub repository (supports private repos)
```

Comment:
I feel like this output may be unexpected? I wasn't expecting the detailed information about `add` to be here at `azd template source --help`

Reply:
updated